### ISSUE TITLE
Renaming handler parameters causes hang on start in v1.1.3

### ISSUE THREAD

**Issue by nxht** [elysiajs/elysia#736]
### What version of Elysia.JS is running?

1.1.3

### What platform is your computer?

Linux 5.15.153.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

```ts
import { Elysia } from 'elysia';

const app = new Elysia().onRequest(({ set: a }) => {});
console.log('start');
app.listen(3000);
```

### What is the expected behavior?

`start` printed and app runs

### What do you see instead?

No output and the web server doesn't run

### Additional information

This cause plugins that uses bun build / bundler like tobias-kaerst-software/elysia-helmet to hang.
Also, the issue is solved on downgrade to 1.1.2

**Comment by cholasimmons** [comment]
I can confirm, Elysia v1.1.3 does not initialize if you're using `.use(helmet())`

**Comment by nxht** [comment]
The issue seems to be fixed in 1.1.4
